### PR TITLE
[Update] API Changelog entry links

### PR DESCRIPTION
**Pull request by leslitagordita**
Update API Changelog entry links to point to new API site

### CODE REVIEW

File: src/content/changelog/2019-10-07-api.md [hunk 1]
```diff
@@ -9,19 +9,19 @@ version: 4.6.0
...
+* The List Managed Stats endpoint [GET managed/stats](https://www.linode.com/docs/api/managed/#managed-stats-list) returns a list of Managed Stats on your Account in the form of x and y data points. You can use these data points to plot your own graph visualizations. These stats reflect the last 24 hours of combined usage across all managed Linodes on your account giving you a high-level snapshot of data for `cpu`, `disk`, `swap`, `network in`, and `network out`.
 
 ### Fixed
 
-* Fixed a formatting error that prevented the [filtering and sorting](https://developers.linode.com/api/v4/#filtering-and-sorting) examples within the API documentation from correctly working as presented.
+* Fixed a formatting error that prevented the [filtering and sorting](https://www.linode.com/docs/api/) examples within the API documentation from correctly working as presented.
```
Comment (hzoppetti) [inline src/content/changelog/2019-10-07-api.md]: can we make this link: http://www.linode.com/docs/api/#filtering-and-sorting

File: src/content/changelog/4-61-0-2020-04-06.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ changelog:
...
 
-- The [Images collection](/api/v4/images-image-id) now contains the `updated` property. This property returns the date that the current Image was last updated.
+- The [Images collection](https://www.linode.com/docs/api/images/#image-view) now contains the `updated` property. This property returns the date that the current Image was last updated.
 
 ### Changed
 
-- The [StackScript collection's](/api/v4/linode-stackscripts-stackscript-id) `deployments_total` property is now filterable. See [Filtering and Sorting](/api/v4/#filtering-and-sorting) to learn how to search API v4 collections.
+- The [StackScript collection's](https://www.linode.com/docs/api/stackscripts/#stackscript-view) `deployments_total` property is now filterable. See [Filtering and Sorting](/https://www.linode.com/docs/api/) to learn how to search API v4 collections.
```
Comment (hzoppetti) [inline src/content/changelog/4-61-0-2020-04-06.md]: same here: http://www.linode.com/docs/api/#filtering-and-sorting

File: src/content/changelog/4-62-0-2020-04-10.md [hunk 3]
```diff
@@ -10,22 +10,22 @@ changelog:
...
+    - List Kubernetes Clusters ([GET /lke/clusters](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-clusters-list))
+    - Create Kubernetes Clusters ([POST /lke/clusters](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-cluster-create))
+    - View Kubernetes Cluster ([GET /lke/clusters/{clusterId}](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-cluster-view))
 
 ### Fixed
 
-- An [LKE cluster's](/api/v4/lke-clusters) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/api/v4/#filtering-and-sorting). These properties have been updated in the [Linode API v4 schema](/api/v4) to no longer display the `filterable` label.
+- An [LKE cluster's](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-clusters-list) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/https://www.linode.com/docs/api/). These properties have been updated in the [Linode API v4 schema](https://www.linode.com/docs/api) to no longer display the `filterable` label.
```
Comment (hzoppetti) [inline src/content/changelog/4-62-0-2020-04-10.md]: and here: http://www.linode.com/docs/api/#filtering-and-sorting

**Review by hzoppetti — CHANGES_REQUESTED** [review]
just a few links for the filter section on the main info page

File: src/content/changelog/2019-10-07-api.md (hunk 1 above)
Comment (leslitagordita) [reply src/content/changelog/2019-10-07-api.md]: yes! Thank you for reviewing...pushed those three things up.

File: src/content/changelog/4-61-0-2020-04-06.md (hunk 2 above)
Comment (leslitagordita) [reply src/content/changelog/4-61-0-2020-04-06.md]: pushed!

File: src/content/changelog/4-62-0-2020-04-10.md (hunk 3 above)
Comment (leslitagordita) [reply src/content/changelog/4-62-0-2020-04-10.md]: pushed!

**Review by hzoppetti — APPROVED** [review]

**Pull request merged**